frontend/spec/views/spree/checkout/_summary_spec.rb

```diff
@@ -5,7 +5,7 @@
   it 'does not use the @order instance variable' do
     order = build(:order)
     expect do
-      render partial: 'spree/checkout/summary', locals: { order: order }
+      render partial: 'spree/checkout/summary', locals: { order: order, classes: '' }
```

Comment:
Do we still need this change?

Reply:
No we don't need it. I've must missed it.